docs/user-guide/aes-operator.md

```diff
@@ -0,0 +1,127 @@
+complete lifecycle of your Ambassador installation in your cluster. It also
+automates many of the repeatable tasks you have to perform for the Ambassador
+Edge Stack. Once installed, the AES Operator will automatically complete rapid
+installations, seamless upgrades to new versions of Ambassador, Azure
+integrations, and provides a reference architecture for ease of use. [Read more](https://github.com/datawire/ambassador-operator/blob/master/README.md#version-syntax).
+
+A Kubernetes operator is a software extension that makes it easier to manage and automate your Kubernetes-based applications, in the spirit of a human operator. Operators complete actions such as deploying, upgrading and maintaining applications. and many others. Read more about Kubernetes Operators [here](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+
+This document covers the configuration and the installation of the Operator for:
+
+* The Ambassador Edge Stack
+* Helm chart
+
+## Install the Operator
+
+Start by installing the operator:
+
+1. Create the Operator Custom Resource schema with the following command: `kubectl apply -f https://github.com/datawire/ambassador-operator/releases/download/latest/ambassador-operator-crds.yaml`
+2. Install the actual CRD for the Ambassador Operator in the `ambassador` namespace with the following command: `kubectl apply -n ambassador -f https://github.com/datawire/ambassador-operator/releases/download/latest/ambassador-operator.yaml`
+3. To install the Ambassador Operator CRD in a different namespace, you can specify it in `NS` and then run the following command:
+
+    ```shell
+    $ NS="custom-namespace"
+    $ curl -L https://github.com/datawire/ambassador-operator/releases/download/latest/ambassador-operator.yaml | \
+        sed -e "s/namespace: ambassador/namespace: $NS/g" | \
+        kubectl apply -n $NS -f -
+    ```
+
+Then, create the `AmbassadorInstallation` Custom Resource schema and apply it to the AES Operator.
+
+1. To create the `AmbassadorInstallation` Custom Resource schema, use the following YAML as your guideline: [https://github.com/datawire/ambassador-operator#the-operator-custom-resource-cr](https://github.com/datawire/ambassador-operator#the-operator-custom-resource-cr).
+2. Save that file as `amb-install.yaml`
+3. Edit the `amb-install.yaml` and optionally complete configurations such as Version constraint or UpdateWindow:
+4. Finally, apply your `AmbassadorInstallation` CRD to the AES Operator schema
+   with the following command: `kubectl apply -n ambassador -f amb-install.yaml`
+
+### Configuration for the Ambassador Edge Stack
+
+After the initial installation of Ambassador, the Operator will check for updates every 24 hours and delay the update until the Update Window allow the update to proceed. It will use the Version Syntax for determining if any new release is acceptable. When a new release is available and acceptable, the Operator will upgrade the Ambassador installation.
```

Comment:
Does it install Ambassador (OSS) or The Ambassador Edge Stack? We should check for consistency regarding "Ambassador" vs. "The Ambassador Edge Stack" throughout here.

Reply:
I would just say Ambassador...